browser-extension/common/src/custom-elements/session-recording-widget/index.ts

```diff
@@ -26,6 +27,7 @@ class RQSessionRecordingWidget extends HTMLElement {
 
   connectedCallback() {
     this.addListeners();
+    super.addDragListeners();
```

Comment:
We should call `super.connectedCallback()` instead of calling `addDragListeners`. It should automatically do all the listeners that is needed to be added